mailpoet/readme.txt

```diff
@@ -12,6 +12,14 @@ Send beautiful newsletters from WordPress. Collect subscribers with signup forms
 
 == Description ==
 
+= Get 40% off all annual plans and upgrades =
+
+Get 40% off all MailPoet annual plans and upgrades – no coupon required. Offer starts at 2 pm UTC on Wednesday, June 14, 2023, and ends at 2 pm UTC on Monday, June 19, 2023. Terms and conditions apply.
+
+[Pick a plan and save](https://account.mailpoet.com/?billing=yearly&ref=sale-june-2023-wporg&utm_source=wordpress.org&utm_medium=description&utm_campaign=sale_june_2023)
+
+= All features =
```

Comment:
@NeosinneR Should this be `What is MailPoet?`?

Reply:
Copy pasting 🤦 Fixed in #994c178